dev/io.openliberty.jakarta.el.4.0/src/jakarta/el/ImportHandler.java

```diff
@@ -508,4 +508,4 @@ public Boolean run() {
      */
     private static class NotFound {
     }
-}
+}
```

Comment:
Should this file be deleted as well? I see the other classes in this project were.

Reply:
This fine has been modified. The line "return cl.getResource(path) == null;" creates a security warning, so I applied a java 2 security check around it.